whatrecord/parse.py

```diff
@@ -74,7 +75,20 @@ def parse(
     """
     standin_directories = standin_directories or {}
 
-    filename = pathlib.Path(filename)
+    if str(filename) == "-":
+        contents = sys.stdin.read()
+        filename = pathlib.Path("/dev/stdin")
```

Comment:
I suppose this is unix-only then?  I'm not sure how to make it work on windows, but I'm sure it's not too big a deal for now.

Reply:
The `/dev/stdin` isn't so meaningful, more of a user-facing name to use for context: e.g., "in `/dev/stdin` on line 5, there was a linter warning"

Your comment made me think I should be using something like `sys.stdin.name` instead, though. That shows up as `<stdin>` on macOS, at least.

Addressed in d843fd5